public/data/bhavyawahie.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Bhavya Wahie",
+    "bio": "Founder of EddieHub",
+    "avatar": "https://github.com/bhavyawahie.png",
+    "links": [
+    {
+        "name": "Follow here to see me talk in memes😎!",
+        "url": "https://twitter.com/bhavyawahie",
+        "icon": "twitter"
+    }, 
+    {
+        "name": "Follow and See my recent side projects on GitHub👩‍💻! ",
```

Comment:
@Bhavyawahie Please add a space between the emoji and the word `GitHub`. Thank you